vega_sim/parameter_test/parameter/loggers.py

```diff
@@ -174,3 +173,43 @@ def tau_scaling_additional_data(
         "Market Open Interest": int(market_data.open_interest)
         / 10**market_info.position_decimal_places
     }
+
+
+def limit_order_book(
+    vega: VegaServiceNull,
+    agents: List[Agent],
+) -> Dict[str, Any]:
+    mm_agent = [
+        agent
+        for agent in agents
+        if isinstance(agent, (OptimalMarketMakerV2, OptimalMarketMaker))
+    ][0]
+
+    order_book = vega.open_orders_by_market(market_id=mm_agent.market_id)
```

Comment:
if you use `vega.order_status_from_feed(live_only=True)` here instead you should see it run a lot faster

Reply:
Although that said, I think `vega.market_depth` should give you what you want here without having to do the processing at all?

Reply:
Will try both methods to choose which better to use.